<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Chatroom</class>
 <widget class="QWidget" name="Chatroom">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>454</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Chat Room</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="1">
        <widget class="QPushButton" name="ConnectToServer">
         <property name="text">
          <string>Connect</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>IP</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="IPaddress"/>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Port</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="PortNumber"/>
         </item>
        </layout>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="disconnect">
         <property name="text">
          <string>Disconnect</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>NAME</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="userName"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QTextBrowser" name="ChatMessage"/>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_4">
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout">
                 <item>
                  <widget class="QPushButton" name="SendMessage">
                   <property name="text">
                    <string>Send Text</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="sendFile">
                   <property name="text">
                    <string>Send File</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <widget class="QPushButton" name="fileBrowse">
                 <property name="text">
                  <string>File Browse</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QLineEdit" name="Message"/>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QListWidget" name="fileList"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
